source/objects/bodyClass.m

```diff
@@ -895,20 +895,12 @@ function regExcitation(obj, w, direction, rho, g, iH)
                 end
             end
            if obj.yaw.option==1
-                % show warning for passive yaw run with incomplete BEM data
-                BEMdir = sort(obj.hydroData(iH).simulation_parameters.direction);
-                boundDiff = abs([-180 180] - BEMdir([1 end]));
-                if length(BEMdir)<3 || std(diff(BEMdir))>5 || max(boundDiff)>15
-                    warning(['Passive yaw is not recommended without BEM data spanning a full yaw rotation -180 to 180 dg.' newline ...
-                        'Please inspect BEM data for gaps'])
-                    clear BEMdir
-                end % wrap BEM directions -180 to 180 dg, if they are not already there
-                obj.hydroForce.(hfName).wrappedDirection = wrapTo180(obj.hydroData(iH).simulation_parameters.direction);
-                obj.makeSpanNeg180To180(iH,hfName);
-                [sortedDir,idx]=sort(obj.hydroForce.(hfName).wrappedDirection);
-                [hdofGRD, hdirGRD, hwGRD] = ndgrid(1:6, sortedDir, obj.hydroData(iH).simulation_parameters.w);
+                % wrap BEM directions -180 to 180 dg, if they are not already there
+                obj.makeSpanNeg180To180(iH);
+                [obj.hydroForce.(hfName).wrappedDirection,idx] = sort(wrapTo180(obj.hydroData(iH).simulation_parameters.direction));
+                [hdofGRD, hdirGRD, hwGRD] = ndgrid(1:obj.dof, obj.hydroForce.(hfName).wrappedDirection, obj.hydroData(iH).simulation_parameters.w);
```

Comment:
if any of my clean-up causes issues or is confusing, 

previous lines 906-908 and new lines 899-900 is the crux of the change that should allow ``simu.reloadH5Data=1``. New line 900 could even be broken up into just wrapping the hydrodata direction and then adding old line 908 back in